lang/python/django/Makefile

```diff
@@ -21,46 +21,51 @@ PKG_LICENSE:=BSD-3-Clause
 PKG_LICENSE_FILES:=LICENSE LICENSE.python
 PKG_CPE_ID:=cpe:/a:djangoproject:django
 
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-django-$(PKG_VERSION)
+
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
+include ../python3-package.mk
```

Comment:
Would also be a great time to update the package as the latest point release has a CVE fix.

Reply:
I would do that in an upcoming series.
There's plenty of things to change here.
And plenty to test & re-test. And I am not quite doing a great job here (yet).

But, if doing things incrementally, I can do a decent job and get better.